src/info.rs

```diff
@@ -78,6 +81,8 @@ pub enum Kind {
     Akp153E,
     /// Ajazz Akl982 Desk Controller
```

Comment:
I know it's not your code but this comment is egregiously wrong, so please amend it.